src/frontend/Komponenter/Behandling/Modal/HenleggModal.tsx

```diff
@@ -34,9 +46,28 @@ export const HenleggModal: FC<{ behandling: Behandling }> = ({ behandling }) =>
         nullstillIkkePersisterteKomponenter,
         settIkkePersistertKomponent,
     } = useApp();
+    const { vergemål, fullmakt } = personopplysninger;
     const [henlagtårsak, settHenlagtårsak] = useState<EHenlagtårsak>();
+    const [huketAvSendBrev, settHuketAvSendBrev] = useState<boolean>(true);
     const [låsKnapp, settLåsKnapp] = useState<boolean>(false);
     const [feilmelding, settFeilmelding] = useState<string>();
+    const [forhåndsvisningsFeil, settForhåndsvisningsFeil] = useState<string>();
+
+    const visBrevINyFane = () => {
+        axiosRequest<string, null>({
+            method: 'GET',
+            url: `familie-ef-sak/api/behandling/${behandling.id}/henlegg/brev`,
```

Comment:
Hvis dette er en ressurs for å hente forhåndsvisning av et brev - burde det blitt reflektert i urlen? 

`familie-ef-sak/api/behandling/${behandling.id}/henlegg/brev/forhandsvisning` ?

Reply:
Tenkte også på dette 👍